src/models/backup.ts

```diff
@@ -64,3 +64,160 @@ class Dropbox {
+            xhr.onreadystatechange = () => {
+              if (xhr.readyState === 4) {
+                if (xhr.status === 401) {
+                  localStorage.removeItem('driveToken');
+                  resolve(false);
+                }
+                try {
+                  const res = JSON.parse(xhr.responseText);
+                  if (res.error) {
+                    if (res.error.code === 404) {
+                      localStorage.driveFolder = '';
+                      resolve(true);
+                    }
+                  } else if (res.trashed) {
+                    localStorage.driveFolder = '';
+                    resolve(true);
+                  } else if (res.error) {
+                    console.error(res.error.message);
+                    resolve(false);
+                  } else {
+                    resolve(true);
+                  }
+                } catch (error) {
+                  console.error(error);
+                  reject(error);
+                }
+              }
+              return;
+            };
+            xhr.send();
+          });
+    }
+    if (!localStorage.driveFolder) {
+      await new Promise(
+          (resolve: (value: boolean) => void,
+           reject: (reason: Error) => void) => {
+            // create folder
+            const xhr = new XMLHttpRequest();
+            xhr.open('POST', 'https://www.googleapis.com/drive/v3/files/');
+            xhr.setRequestHeader('Authorization', 'Bearer ' + token);
+            xhr.setRequestHeader('Accept', 'application/json');
+            xhr.setRequestHeader('Content-Type', 'application/json');
+            xhr.onreadystatechange = () => {
+              if (xhr.readyState === 4) {
+                if (xhr.status === 401) {
+                  localStorage.removeItem('driveToken');
+                  resolve(false);
+                }
+                try {
+                  const res = JSON.parse(xhr.responseText);
+                  if (!res.error) {
+                    localStorage.driveFolder = res.id;
+                    resolve(true);
+                  } else {
+                    console.error(res.error.message);
+                    resolve(false);
+                  }
+                } catch (error) {
+                  console.error(error);
+                  reject(error);
+                }
+              }
+              return;
+            };
+            xhr.send(JSON.stringify({
+              'name': 'Authenticator Backups',
+              'mimeType': 'application/vnd.google-apps.folder'
+            }));
+          });
+    }
+    return localStorage.driveFolder;
+  }
+
+  async upload(encryption: Encryption) {
+    if (localStorage.driveEncrypted === undefined) {
+      localStorage.driveEncrypted = 'true';
+    }
+    const exportData = await EntryStorage.getExport(
+        encryption, (localStorage.driveEncrypted === 'true') ? true : false);
```

Comment:
`localStorage.driveEncrypted === 'true'` is the same as `(localStorage.driveEncrypted === 'true') ? true : false`